python/seldon_core/utils.py

```diff
@@ -397,8 +463,12 @@ def construct_response(user_model: SeldonComponent, is_request: bool, client_req
         raise SeldonMicroserviceException("Unknown data type returned as payload:" + client_raw_response)
 
 
-def extract_request_parts_json(request_raw: Union[Dict, List]) -> Tuple[
-    Union[np.ndarray, str, bytes, dict], Dict, prediction_pb2.DefaultData, str]:
+def extract_request_parts_json(request: Union[Dict, List]
```

Comment:
This PR looks good to me. 
Probably not related to this PR, but I was wondering why gRPC path should not go this way. 
From my profiler commented in issue it shows in gRPC the input was first converted to `datadef` in `SeldonMessage`:

```
100    0.240    0.002    1.281    0.013 utils.py:234(array_to_grpc_datadef)
20100/100    0.038    0.000    1.041    0.010 utils.py:280(array_to_list_value)
```

then in `predict` function it is converted back to Numpy array:

```
100    0.000    0.000    2.781    0.028 utils.py:505(extract_request_parts)
100    0.001    0.000    2.778    0.028 utils.py:120(get_data_from_proto)
100    0.001    0.000    2.778    0.028 utils.py:147(grpc_datadef_to_array)
```

The round-trip conversion seems double the latency I guess?